onboarding_project/squad_pantry_app/admin.py

```diff
@@ -180,6 +184,20 @@ class SquadUserAdmin(UserAdmin):
     filter_horizontal = ('user_permissions', 'groups',)
 
 
+class ConfigurationSettingsAdmin(admin.ModelAdmin):
```

Comment:
This is not what we discussed. We had two ideas:

1. Have a `Settings` model like this:
```py
class Settings(models.model):
    order_limit = model.IntegerField()
    another_setting = model.CharField()
    ....
    <other_settings>
```
And create a single instance of the above model and do not let any one add more records. If you want add another setting, you add a new field to this model and migrate.

2. Have a configurable model:
```py
class ConfigurationSettings(models.model)
     key = models.CharField()
     value = models.CharField()
```

And create a `limit` = `<num>` key value pair. In this case, the admin must be able to add new records. How else would you be able to specify a new setting?

What you have done is a mixture of method 1 and 2.

Reply:
Done!